js/docEnums.js

```diff
@@ -384,6 +384,11 @@
  * @enum {'empty'|'group'|'simple'|'tabbed'}
  */
 
+ /**
+ * @typedef {string} Enums.FormItemAlignment
```

Comment:
@San4es Is it a good name for an MVC enum?

Reply:
LGTM